packages/flutter/lib/src/gestures/recognizer.dart

```diff
@@ -324,9 +358,17 @@ abstract class PrimaryPointerGestureRecognizer extends OneSequenceGestureRecogni
   @override
   void handleEvent(PointerEvent event) {
     assert(state != GestureRecognizerState.ready);
-    if (state == GestureRecognizerState.possible && event.pointer == primaryPointer) {
-      // TODO(abarth): Maybe factor the slop handling out into a separate class?
-      if (event is PointerMoveEvent && _getDistance(event) > kTouchSlop) {
+    if (event.pointer == primaryPointer) {
```

Comment:
Why's the state check no longer required?

Reply:
Oh, is this because we want to track movements after the gesture was accepted and the state is defunct at that point?

Reply:
No, I shouldn't have removed it. Thanks for catching.